frontend/docs/components/code/CodeBlock.tsx

```diff
@@ -3,65 +3,125 @@ import { parseDocComments } from "./codeParser";
 import { Src } from "./codeData";
 import CodeStyleRender from "./CodeStyleRender";
 import { Button } from "../ui/button";
-import { CheckIcon, CopyIcon, FoldVertical, MoveRight, MoveUpRight, UnfoldVertical } from "lucide-react";
+import {
+  CheckIcon,
+  CopyIcon,
+  FoldVertical,
+  MoveRight,
+  MoveUpRight,
+  UnfoldVertical,
+} from "lucide-react";
 
 interface CodeRendererProps {
   source: Src;
   target: string;
 }
 
 export const CodeBlock = ({ source, target }: CodeRendererProps) => {
-    const [collapsed, setCollapsed] = React.useState(true);
-    const [plainText, setPlainText] = React.useState(false);
-    const [copied, setCopied] = React.useState(false);
+  const [collapsed, setCollapsed] = React.useState(true);
+  const [plainText, setPlainText] = React.useState(false);
+  const [copied, setCopied] = React.useState(false);
 
-    const parsed = parseDocComments(source.raw, target, collapsed);
+  const parsed = parseDocComments(source.raw, target, collapsed);
 
-    return <>
-        <div className="sticky top-0 z-10 bg-background flex flex-row gap-4 justify-between items-center pl-2 mb-2">
-            <a href={source.githubUrl} target="_blank" rel="noopener noreferrer" className="text-xs text-gray-500 font-mono hover:underline">{source.props.path}</a>
-            <div className="flex gap-2 justify-end">
-                <Button variant="ghost" size="sm" onClick={() => setCollapsed(!collapsed)}>
-                    {collapsed ? <><UnfoldVertical className="w-4 h-4 mr-2" />Expand</> : <><FoldVertical className="w-4 h-4 mr-2" />Collapse</>}
-                </Button>
-                <Button variant="outline" size="sm" onClick={() => {
-                    navigator.clipboard.writeText(parsed);
-                    setCopied(true);
-                    setTimeout(() => setCopied(false), 2000);
-                }}>
-                    {copied ? <><CheckIcon className="w-4 h-4 mr-2" />Copied</> : <><CopyIcon className="w-4 h-4 mr-2" />Copy</>}
-                </Button>
-            </div>
+  return (
+    <>
+      <div className="sticky top-0 z-10 bg-background flex flex-row gap-4 justify-between items-center pl-2 mb-2">
+        <a
```

Comment:
nit but a thing for the future - I think it's probably recommended to use `Link` from `next/link` here instead of an `a` tag, but this is fine 👍

Reply:
actually just ran `prettier` here, so this is really a 1-line change on the docs